packages/eslint-plugin/tests/rules/explicit-function-return-type.test.ts

```diff
@@ -328,29 +328,31 @@ const func = (value: number) => x as const;
       filename: 'test.ts',
       code: `
 function test() {
-    return;
+  return;
 }
       `,
       errors: [
         {
           messageId: 'missingReturnType',
           line: 2,
           column: 1,
+          endColumn: 15,
```

Comment:
I know it'll be annoyingly verbose, but for sanity checking, could you please also add `endLine` to these tests.

I just wanna be 100% sure that nobody accidentally does something weird to break it in future :)

Reply:
@bradzacher Sure :+1: